scripts/init.js

```diff
@@ -23,9 +23,14 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
     appPackage.dependencies[key] = ownPackage.devDependencies[key];
   });
 
+  // Test config
+  appPackage.jest = {
+    preset: '<rootDir>/node_modules/react-scripts/config/jest',
```

Comment:
So this is writing one line of configuration into the `package.json`. Kind of aesthetic but we manage to avoid it for webpack, babel, & eslint. Is there a way around it here? also curious for @gaearon thoughts if there is a way around this.